examples/gpg/gpg_keys.sh

```diff
@@ -30,6 +30,7 @@ Key-Type: RSA
 Key-Length: 2048
 Subkey-Type: RSA
 Subkey-Length: 2048
+Preferences: SHA256 SHA384 SHA512 SHA224 AES256 AES192 AES CAST5 ZLIB BZIP2 ZIP Uncompressed
```

Comment:
What is the requirement for listing CAST5 here?

Also, is the order of algorithms important here? I wonder why you list the SHA384 (truncated SHA512) first but the SHA224 (truncated SHA256) last.